content/en/cosign/signing_with_blobs.md

```diff
@@ -1,71 +1,24 @@
 ---
-title: "Working with Blobs"
+title: "Signing Blobs"
 category: "Cosign"
 position: 109
 ---
 
-Cosign supports signing and verifying standard files and blobs (or binary large objects), in addition to containers.
+Cosign supports signing and verifying standard files and blobs (or binary large objects), in addition to containers. This topic discusses signing.  For information on verifying see [Verifying](/cosign/verify).
 
-You can upload blobs to an OCI registry (similar to ORAS) where they can then be signed/verified like any other image, or you can sign blobs locally as standard files.
+## Signing blobs as files in keyless mode
```

Comment:
Can we add a definition for "keyless"? Something like:

Cosign supports identity-based signing, associating an ephemeral signing key with an identity from an OpenID Connect provider. We refer to this process as "keyless signing".